src/components/menu-list/menu-list.stories.mdx

```diff
@@ -1,179 +1,164 @@
-import { useState } from 'react';
-import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
-import { MenuList, MenuListItem } from '.'
+import { useState } from "react";
+import { Meta, Story, Preview, Props } from "@storybook/addon-docs/blocks";
+import { MenuList, MenuListItem } from ".";
 
-<Meta title="MenuList" parameters={{ info: { disable: true }}} />
+<Meta title="MenuList" parameters={{ info: { disable: true } }} />
```

Comment:
Should we disable the chromatic across all these stories now it's being deprecated?